docs/application/get-started/mobile.md

```diff
@@ -1,44 +1,49 @@
-# Get Started for Mobile
+# Get Started with Tizen Mobile Applications
 
-The Get Started Tutorial teaches you how to create basic Tizen applications for mobile devices.
+Tizen ecosystem is enhanced with advanced Tizen-based smartphone application development.  
+For mobile devices, Tizen supports three primary application types, .NET, Web, and Native applications.
 
-Tizen supports three primary application types for mobile:
-
--   [.NET application](#dotnet-application) is a new way to develop applications for the Tizen operating system using C# language.
-
--   [Web application](#web-application) is a Web site stored on your device and built using Web-native languages, such as HTML5, CSS, and JavaScript. The Web application uses the Tizen Web framework to interact with the native subsystems.
-
--   [Native application](#native-application) is developed using C language. Developer can use the dozens of native API modules that cover a large range of capabilities such as camera, GPS, and accelerometer in addition to more advanced system settings and functionality.
-
-<a name="dotnet-application"> </a>
-## .NET Application
-
-The following guide demonstrates how to create and run a basic Tizen .NET application, which displays some text on the screen with no user interaction:
+## Mobile .NET Applications
+You can create rich applications in C# using the .NET framework benefits that contribute to secure and efficient application development. 
 
+To begin with, you can use the following quick start guide:
```

Comment:
Is there any clear phrase?

Reply:
Can we rephrase as:
To begin your Tizen .NET application development journey, use the following quick start guide:

Please verify and share your valuable feedback. 
If you agree with this, I will update all the three files.